mobs/nakki_v2/NakkiV2.cs

```diff
@@ -0,0 +1,291 @@
+using Godot;
+using System;
+using System.Linq;
+
+using Jakojaannos.WisperingWoods.Characters.Player;
+using Jakojaannos.WisperingWoods.Util.Editor;
+
+namespace Jakojaannos.WisperingWoods;
+
+[Tool]
+public partial class NakkiV2 : Path2D {
+	public override string[] _GetConfigurationWarnings() {
+		var warnings = base._GetConfigurationWarnings() ?? Array.Empty<string>();
```

Comment:
I think shorthand collection initialization is available due to bump to .NET8
```suggestion
		var warnings = base._GetConfigurationWarnings() ?? [];
```